pkg/ctl/utils/describe_stacks.go

```diff
@@ -48,14 +52,15 @@ func describeStacksCmd(cmd *cmdutils.Cmd) {
 				return err
 			}
 		}
-		return doDescribeStacksCmd(cmd, all, events, trail, printer)
+		return doDescribeStacksCmd(cmd, all, events, trail, resourceStatus, printer)
 	}
 
 	cmd.FlagSetGroup.InFlagSet("General", func(fs *pflag.FlagSet) {
 		cmdutils.AddClusterFlagWithDeprecated(fs, cfg.Metadata)
 		cmdutils.AddRegionFlag(fs, &cmd.ProviderConfig)
 		fs.BoolVar(&all, "all", false, "include deleted stacks")
 		fs.BoolVar(&events, "events", false, "include stack events")
+		fs.StringSliceVar(&resourceStatus, "resource-status", nil, "resource statuses to filter events by, e.g. `CREATE_FAILED`, `UPDATE_FAILED`")
```

Comment:
How about naming this flag `--events-filter|type`? do we already use `--resource-status` in other places?

Reply:
I don't think we use `--resource-status` in any other places. We could use your suggested approach if we find other fields of interest to filter by, I guess. I'll merge the PR as is for now, but I'm open to changes for future improvements.